zebra-scan/src/tests/vectors.rs

```diff
@@ -146,18 +147,16 @@ async fn scanning_zecpages_from_populated_zebra_state() -> Result<()> {
 ///
 /// The purpose of this test is to check if our database and our scanning code are compatible.
 #[test]
-#[allow(deprecated)]
 fn scanning_fake_blocks_store_key_and_results() -> Result<()> {
+    let network = Network::Mainnet;
+
     // Generate a key
-    let extsk = ExtendedSpendingKey::master(&[]);
-    // TODO: find out how to do it with `to_diversifiable_full_viewing_key` as `to_extended_full_viewing_key` is deprecated.
-    let extfvk = extsk.to_extended_full_viewing_key();
-    let dfvk: DiversifiableFullViewingKey = extsk.to_diversifiable_full_viewing_key();
-    let key_to_be_stored =
-        zcash_client_backend::encoding::encode_extended_full_viewing_key("zxviews", &extfvk);
+    let efvk = mock_sapling_efvk(&[]);
+    let dfvk = efvk.to_diversifiable_full_viewing_key();
+    let key_to_be_stored = encode_extended_full_viewing_key("zxviews", &efvk);
 
     // Create a database
-    let mut s = new_test_storage(Network::Mainnet);
+    let mut s = new_test_storage(network);
```

Comment:
Nitpick: This wasn't in this PR, and it's a test, but we should try to avoid single-letter variable names outside short closures.

Reply:
Addressed in [5118f76](https://github.com/ZcashFoundation/zebra/pull/8251/commits/5118f76a9199c2158f20fe9e91c915e538fcc88e).